### PR TITLE
Drone stuffs

### PR DESCRIPTION
**Pull request by 70000hp**
**Added**
-   All around drone speed improvements, Logistics drones got slightly boosted, transport got heavily boosted
-   Drones can now work on low angles
-   Breaking drones while flying actually yields the drone, and the resources its carrying
-   Fixed Logistics drones  getting stuck
-   Added transport drone visualization
-   Heavily reduced the logistics drone spawning delay
-   Added Redstone support for providers and requesters, which allows you to disable them

### CODE REVIEW

**Comment by signed-firmware** [comment]
omg 7k stopped yapping and actually did shit 😨 😱

**Comment by signed-firmware** [comment]
becides the absurb yapenseium atleast we got good drones

**Comment by Vaern** [comment]
god gives his strongest battles to 7's greatest haters

File: src/main/java/com/hbm/entity/item/EntityDroneBase.java [hunk 1]
```diff
@@ -112,7 +112,9 @@ public void onUpdate() {
 					this.motionZ = dist.zCoord * speed;
 				}
 			}
-			
+			if(isCollidedHorizontally){
+				motionY += 1;
```
Comment (HbmMods) [inline src/main/java/com/hbm/entity/item/EntityDroneBase.java:116]: wouldn't this cause drones to shoot up uncontrollably when smacking into the side of a tall wall?

Comment (legendarydoge30) [reply src/main/java/com/hbm/entity/item/EntityDroneBase.java:116]: its an amazing pathfinding ai

Comment (70000hp) [reply src/main/java/com/hbm/entity/item/EntityDroneBase.java:116]: Oh yes, but if your drone is running into a gigantic wall, there is something VERY wrong with your setup
In 99% of cases it fixes that stupid bug, and in the 1% it has a harmless side effect

**Comment by 70000hp** [comment]
> omg 7k stopped yapping and actually did shit 😨 😱

how about you start doing shit you gibbon

**Comment by signed-firmware** [comment]
no i suck at coding 👍

**Pull request merged**